packages/config/config/devices/0x013c/pst02a.json

```diff
@@ -61,47 +60,309 @@
 				}
 			]
 		},
-		"5": {
-			"label": "Operation Mode",
-			"description": "Mode of operation and enabled multisensor functions",
+		"5[0x02]": {
+			"label": "Test Mode",
 			"valueSize": 1,
 			"minValue": 0,
-			"maxValue": 127,
+			"maxValue": 1,
 			"defaultValue": 0,
+			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "Set Celsius",
-					"value": 8
+					"label": "Disable",
+					"value": 0
 				},
 				{
-					"label": "Preset: Celsius and LED on = Bits: 00001010 = 10",
-					"value": 10
+					"label": "Enable",
+					"value": 1
 				}
 			]
 		},
-		"6": {
-			"label": "Multi-Sensor Function Switch",
-			"description": "Enable or disable functions of the multisensor.",
+		"5[0x04]": {
+			"label": "Door/Window Function",
 			"valueSize": 1,
 			"minValue": 0,
-			"maxValue": 127,
-			"defaultValue": 4
```

Comment:
The old parameter 6 had a default value of 4, so 6[0x04] should have a default value of 1, the rest 0 (if I'm not mistaken).

Reply:
Change default value to 0 for 6[0x20]